### ISSUE TITLE
Set Default build task by default

### ISSUE THREAD

**Issue by jacoues** [vshaxe/vscode-terminator#4]
In order to minimize the number of things to edit when I switch default build tasks. I found that putting "args": [${defaultBuildTask}] runs the default build task as defined in tasks.json. This means I only have to edit tasks.json in order to switch default build task.
Perhaps you could add that to your documentation?

**Comment by Gama11** [comment]
Interesting, I didn't know about that!

**Comment by Gama11** [comment]
`"args": [${defaultBuildTask}]` isn't valid syntax, so I assume you meant:

```
"args": ["${defaultBuildTask}"]
```

However, that doesn't seem to work:

![](https://i.imgur.com/KFnYbIU.png)

**Comment by jacoues** [comment]
You're right.. for some reason though, despite the invalid syntax it still runs the default task. Perhaps if the task is unspecified or invalid it runs the default build task?

**Comment by Gama11** [comment]
That's strange, that's not the case for me. It doesn't make sense when you look at the code either:

https://github.com/vshaxe/vscode-terminator/blob/ec59075fb507f3249f3be9773afd83b38cf29262/src/Main.hx#L28-L36

The error is shown in the `if`, `executeTask()` is called in the `else`.
